### DISCUSSION TITLE
Items For Luck-Todo

### DISCUSSION THREAD

**Discussion by szdytom** [segment-oj/segmentoj-frontend#151] [Ideas]
Let's think some items for the luck-todolist on the homepage.

**Comment by szdytom** [comment]
**Worship FISHER_**: RP++
**膜拜神渔**: RP++

**Comment by szdytom** [comment]
**Stay up late**: I can solve it!
**熬夜**: 我能完成的！

**Comment by szdytom** [comment]
**Refactory**: clean my codes up.
**重构代码**: 整理好我的代码

**Comment by szdytom** [comment]
**Learn C++**: https://en.cppreference.com
**学习C++**: https://zh.cppreference.com

**Comment by ZTL-UwU** [comment]
**M-Fight / 2048**: one last time.
**M-Fight / 2048**: 再来最后一把

**Comment by szdytom** [comment]
**Hack OJ**: find a bug.
**Hack OJ**: 找个漏洞

**Comment by ZTL-UwU** [comment]
**AC**: n more problems.
**AC**: 再切 n 道题

**Comment by szdytom** [comment]
**Blog**: share what I learned.
**博客**: 分享知识

**Comment by ZTL-UwU** [comment]
**apt**: `sudo apt update && apt upgrade`.

**Comment by ZTL-UwU** [comment]
**Light**: close the light before leaving.
**灯**: 走前记得关

**Comment by ZTL-UwU** [comment]
**Lock**: lock my account !!!
**锁号**: 防止 JC

**Comment by szdytom** [comment]
**Dig Holes**: find 'The book of AK IOI'
**寻宝**: 找到《10分钟速成AK IOI》

**Comment by szdytom** [comment]
`passwd root`: ubuntu is so safe.

**Comment by ZTL-UwU** [comment]
**rm**: `rm -rf /*` 360 deep cleaning.
**rm**: `rm -rf /*` 360 深度清理

**Reply by szdytom** [reply]
这样也许会更好：
**clean rubbish**: `sudo rm -rf /*`
**清理垃圾**: `sudo rm -rf /*`

**Comment by ZTL-UwU** [comment]
**SOJ**: star SOJ
**SOJ**: 给 SOJ 一个star
